jest.config.js

```diff
@@ -15,7 +15,7 @@ module.exports = {
         '!./js/renovation/list.tsx',
         '!./js/renovation/utils/render-template.tsx',
     ],
-    coverageDirectory: './testing/jest/code_coverage',
+    coverageDirectory: './js/renovation/code_coverage',
     coverageThreshold: {
         './js/renovation/**/*.tsx': {
```

Comment:
j.tsx files are taken into account when coverage is calculated. I think we should exclude them from coverage.